modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManagerSelfTest.java

```diff
@@ -581,6 +581,40 @@ public void testSnapshotThreadPoolSizeUsage() throws Exception {
         assertEquals(snapshotThreadPoolSize.longValue(), snpRunningThreads);
     }
 
+    /**
+     * Tests that full-copy snapshot logs correctly.
+     *
+     * @throws Exception If fails.
+     * */
+    @Test
+    public void testSnapshotCreationLog() throws Exception {
+        if (listenLog == null)
```

Comment:
You don't need this check. `listenLog` must be new for every test to avoid side effects.